CHANGES.rst

```diff
@@ -25,6 +25,6 @@
 Changes
 =======
 
-Version 1.0.0 (released 2016-01-07)
+Version 1.0.0 (released 2016-02-05)
```

Comment:
I was adding it everywhere in recent releases. I think we talked about it with @tiborsimko.